src/helpers/runes.ts

```diff
@@ -0,0 +1,5 @@
+import { Rune } from "@/types";
+
+export const runesToString = (runes: Rune[]): string => {
```

Comment:
Can you change it to a (Rune | string)[] and just use the string if it is given.